libs/agora/api-description/src/paths/teams.yaml

```diff
@@ -0,0 +1,17 @@
+get:
+  tags:
+    - Teams
+  summary: Get Teams
+  description: Get Teams
+  operationId: getTeams
```

Comment:
In OC, we use this convention:

Get multiple entities: listTeams
Get single entity: getTeam

The OpenAPI Generator use the operationId to generate the name of the service methods in the client. I find it easier to distinct when one or multiple entities are fetched when we use "getX" and "listX" compared to "getX" and "getXs".

Reply:
I like that convention since plurals in English don't always have 's' at the end.

Reply:
will update the operationId to listTeams